README.md

```diff
@@ -83,32 +94,25 @@ git clone https://github.com/Azure/Moneo.git
 cd Moneo
 
 # install dependencies
-python3 -m pip install ansible
+sudo apt-get install pssh
```

Comment:
Do we need to fix pssh version here as well?

Reply:
good catch